Makefile

```diff
@@ -22,8 +22,19 @@ init:
 
 	until docker exec $(POSTGRES_CONTAINER_NAME) pg_isready ; do sleep 5 ; done
 
+	sleep 5
+
+	until docker exec $(POSTGRES_CONTAINER_NAME) pg_isready ; do sleep 5 ; done
+
 	docker exec $(POSTGRES_CONTAINER_NAME) sh -c "psql -U postgres -d heaven_in_mouth < db_setup.sql"
 
+.PHONY: rebuild
+rebuild: 
+	make stop
+	docker compose down --remove-orphans
+	sudo rm -rf /var/tmp/heaven_in_mouth_db_data/*
+	make init
+
```

Comment:
I like that, but it can be improved a bit. We can do it like that: 
## Create two `targets`: 
1.  **cleanup** -> which will do `docker-compose down --remove-orphans` and `rm -rf /var/tmp/heaven_in_mouth_db_data/` (note. sudo and /* and make stop is not needed here -> docker-compose stop is weaker than down so it doesn't matter if we run it -> down will remove everything). 
2. **rebuild** -> than we can create rebuild target with use of 2 sources (`cleanup` and `init`). The `rebuild` target would look like this:
```Makefile
.PHONY: rebuild
rebuild: cleanup, init
```
This `rebuild` will execute all commands from `cleanup` and after executing `cleanup` will execute `init` commands. For reference please read this short 3min article about fundamentals of Makefile [link](https://www.digitalocean.com/community/tutorials/how-to-use-makefiles-to-automate-repetitive-tasks-on-an-ubuntu-vps#understanding-makefiles)